constrain/library/G36CoolingOnlyTerminalBoxDeadbandAirflowSetpoint.py

```diff
@@ -0,0 +1,74 @@
+"""
+G36 2021
+### Description
+
+Section 5.5.5.2
+
+- When the Zone State is deadband, the active airflow setpoint shall be the minimum endpoint.
+
+Verification Item:
+
+- When in deadband stage, check if active airflow setpoint is near the correct minimum value.
+
+### Verification logic
+
+```
+switch operation_mode
+switch operation_mode
+case 'occupied'
+    minimum = v_min*
+case 'cooldown', 'setup', 'warmup', 'setback', 'unoccupied'
+    minimum = 0
+
+if abs(v_spt - minimum) <= v_spt_tol
+    pass
+else
+    fail
+end
+```
+
+### Data requirements
+
+- operation_mode: System operation mode
+- zone_state: Zone state (heating, cooling, or deadband (not in either heating or cooling))
+- v_min*: Occupied zone minimum airflow setpoint
+- v_spt: Active airflow setpoint
+- v_spt_tol: Airflow setpoint tolerance
```

Comment:
I see that we use a tolerance here, do you think we also need to use a tolerance for the other verifications?

Reply:
Yes I think there are in consistencies. I use tolerance when the data range is not 0-100 and where, without tolerance, it will be very difficult to pass. Let's review and align all verifications in a future task.